packages/tse-backend/src/common/utils/subscriptionKey.ts

```diff
@@ -0,0 +1,50 @@
+import { MarketDataType } from '../enums/exchange-data.enums';
+
+export interface CompositeKey {
+  type: MarketDataType;
+  exchange: string;
+  symbol?: string;
+  symbols?: string[];
+  timeFrame?: string;
+}
+
+export const createCompositeKey = (
+  type: MarketDataType,
+  exchange: string,
+  symbol?: string,
+  symbols?: string[],
+  timeFrame?: string,
+): string => {
+  let key = '';
+  if (type === MarketDataType.ORDERBOOK || type === MarketDataType.TICKER) {
+    key = `${type}:${exchange}:${symbol}`;
+  } else if (type === MarketDataType.OHLCV) {
+    key = `${type}:${exchange}:${symbol}:${timeFrame}`;
+  } else if (type === MarketDataType.TICKERS) {
+    key = `${type}:${exchange}:${symbols}`;
+  }
+  return key;
```

Comment:
What's that, and why do we need this?

Reply:
This utility automatically creates and manages unique keys based on subscriber-defined data preferences, simplifying subscription management and data routing in websocket

Reply:
add a code comment explaining it

Reply:
Added: 7fdad0b8